dockerfiles/dataset/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:14.04
```

Comment:
:question: Consider using `ubuntu:16.04` or `python-slim` (which I think is based on Ubuntu; so you should be able to install deb packages on it)